plugins/inputs/gnmi/extensions/jnpr_gnmi_extention/GnmiJuniperTelemetryHeaderExtension.pb.go

```diff
@@ -4,7 +4,7 @@
 //      protoc        v3.12.4
 // source: GnmiJuniperTelemetryHeaderExtension.proto
 
-package jnpr_gnmi_extention
+package jnpr_gnmi_extension
```

Comment:
I'm not sure we should touch this file it literally says do not edit ;)

```suggestion
package jnpr_gnmi_extention
```

Reply:
Reverted.